tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -256,174 +258,177 @@ PRJDIR := $(GENDIR)prj/
 # examples/micro_speech/Makefile.inc for an example. At the same time, we
 # search till an arbitrary depth for files named Makefile_internal.inc as a way
 # to bypass this check and allow for deeper directory structures.
-MICRO_LITE_EXAMPLE_TESTS := $(shell find tensorflow/lite/micro/examples/ -maxdepth 2 -name Makefile.inc)
-# Internal examples are copied outside the TFLM repo in ../google.
-ifneq ($(wildcard ../google),)
-MICRO_LITE_EXAMPLE_TESTS += $(shell find ../google/ -name Makefile_internal.inc)
+MICRO_LITE_EXAMPLE_TESTS := $(shell find $(TENSORFLOW_ROOT)tensorflow/lite/micro/examples/ -maxdepth 2 -name Makefile.inc)
+
+# Internal examples are copied outside the TFLM repo in google.
+ifeq ($(TENSORFLOW_ROOT),)
+  MICRO_LITE_EXAMPLE_TESTS += $(shell find ../google/ -name Makefile_internal.inc)
+else
+  MICRO_LITE_EXAMPLE_TESTS += $(shell find google/ -name Makefile_internal.inc)
 endif
```

Comment:
we should be able to replace with something lilnt

```
ifneq ($(EXTERNAL_INCLUDE_DIR),)
  MICRO_LITE_EXAMPLE_TESTS += $(shell find $(EXTERNAL_INCLUDE_DIR) -name Makefile_internal.inc)
endif
```

Reply:
Done.